client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,68 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getEditorNewPostPath } from 'state/ui/editor/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+// We discard all other props in connect() to make sure the component only receives step objects.
+const TodoLinks = steps =>
+	map( steps, ( { label, url }, stepName ) => (
+		<div key={ stepName } className="steps__summary-entry todo">
+			<a href={ url }>{ label }</a>
+		</div>
+	) );
+
+export default localize(
+	connect(
+		( state, { siteId, siteSlug, siteUrl, translate } ) => {
+			const isConnected = isJetpackSite( state, siteId ); // Will only return true if it's connected to WP.com
+			if ( isConnected ) {
+				return {
```

Comment:
We'll always returning a new object here. Could we save some unnecessary renders by memoizing it when the same props are passed?

Reply:
I thin I'd like to leave the memoization for another PR (since I’d have to think about it a bit, and it’s likely to take a while since it gets into its final shape). Getting a bit drained by how long this stupidly trivial PR is taking, and I’d like to unblock us. You okay with iterating in this case?